.travis.yml

```diff
@@ -56,15 +65,17 @@ install:
   # Deactivate the Qiita environment
   - conda deactivate
   # Set the labcontrol config file
-  - sed "s#CERTIFICATE_FILEPATH=#CERTIFICATE_FILEPATH=$PWD/support_files/server.crt#g" labcontrol/db/support_files/test_config.cfg > ~/.labcontrol.cfg.1
+  - sed "s#CERTIFICATE_FILEPATH=#CERTIFICATE_FILEPATH=$PWD/support_files/server.crt#g" labcontrol/db/support_files/qiita_test_config.cfg > ~/.labcontrol.cfg.1
```

Comment:
Why was this file renamed to "qiita_test_config".cfg?  It holds considerably more than qiita config info ...

Reply:
My bad. You're right, this is a labcontrol config file; I mistook it for a sample Qiita config file. Something other than 'test_config.cfg' might be more appropriate, as it can be interpreted as configuration information for Labcontrol's own tests; what do you think? Perhaps 'sample.cfg' or 'travis_build.cfg' might be more illuminating.

Reply:
It isn't just used for tests.  I don't think I named it, but I suspect it was named "test_config" not because it is related to unit tests, but because it is the exercises (tests) the minimal config settings that one needs to run the software (back before the certificate stuff came in, this file could work as the config without any user modification).  

Now it is sort of a skeleton config, not really a working config on its own.  We could call it labcontrol_config.cfg, if you like, or skeleton_labcontrol_config.cfg, or base_labcontrol_config.cfg, or ....?

Reply:
:+1: on latter, "travis_build.cfg"

Reply:
We *could* call it "sample_config.cfg" (per your suggestion) but I'm vaguely uncomfortable with that because, nowadays, it isn't a working config file right out of the box--the user has to fill in some more values.  Let's definitely stay away from "travis_build.cfg" because this config file is used when the end user sets up the software, not just during travis building.

Reply:
'cfg' itself implies config - maybe removing '_config' might make these a little less wordy:
sample.cfg
labcontrol.cfg
skeleton_labcontrol.cfg
base_labcontrol.cfg

In some of the open-source software I use, it's typical to have these incomplete config files as 'something.conf.sample' - perhaps these might also be a good alternative:

labcontrol.conf.sample
labcontrol.cfg.sample

Reply:
I vote for skeleton_labcontrol.cfg or base_labcontrol.cfg and against the use of the name "sample", which is used all over labcontrol to mean something different than it would mean here (and what it would mean here isn't what I would expect it to mean :)

Reply:
I vote for 'skeleton_labcontrol.cfg' over 'base_labcontrol.cfg', as it better implies that this file is incomplete.

Reply:
Sold!